package.json

```diff
@@ -24,7 +24,7 @@
     "export:measures": "scripts/measures/export-measures",
     "pretest": "eslint test scripts index.js",
     "pre-commit": "tools/pre-commit.sh",
-    "jest:cov": "TS_NODE_FILES=true NODE_OPTIONS='--max-old-space-size=3072' jest --coverage --coverage-prodivder=v8 --runInBand"
+    "jest:cov": "TS_NODE_FILES=true NODE_OPTIONS='--max-old-space-size=3072' jest --coverage --coverage-prodivder=v8"
```

Comment:
Looks like a typo here

Reply:
Fixed. I had to update a bunch of ignore-line commands too, since they were not v8-complient.